### ISSUE TITLE
Error when ./hanlon_init

### ISSUE THREAD

**Issue by dgorack** [csc/Hanlon#374]
While following the [Installation (Overview)](https://github.com/csc/Hanlon/wiki/Installation-%28Overview%29) I get this error at step: Create 'sample' configuration:

Tried on CentOS 6.6 and CentOS 7.0.

```
[root@localhost hanlon]# ./hanlon_init 
NotImplementedError: C extension support is not enabled. Pass -Xcext.enabled=true to JRuby or set JRUBY_OPTS.

   (root) at /root/.rbenv/versions/jruby-1.7.8/lib/ruby/shared/mkmf.rb:8
  require at org/jruby/RubyKernel.java:1084
  require at /root/.rbenv/versions/jruby-1.7.8/lib/ruby/shared/rubygems/core_ext/kernel_require.rb:55
  require at /root/.rbenv/versions/jruby-1.7.8/lib/ruby/shared/rubygems/core_ext/kernel_require.rb:53
   (root) at /opt/hanlon/core/config/server.rb:1
  require at org/jruby/RubyKernel.java:1084
  require at /root/.rbenv/versions/jruby-1.7.8/lib/ruby/shared/rubygems/core_ext/kernel_require.rb:55
  require at /root/.rbenv/versions/jruby-1.7.8/lib/ruby/shared/rubygems/core_ext/kernel_require.rb:53
   (root) at ./hanlon_init:18
```

bundle install completes successfully.

Install jruby via rbenv

    # ruby -v
    jruby 1.7.8 (1.9.3p392) 2013-11-14 0ce429e on OpenJDK 64-Bit Server VM 1.6.0_34-b34 [linux-amd64]

    # bundler -v
    Bundler version 1.9.2

I also get `Your Ruby version is 1.9.3, but your Gemfile specified 2.2.1` on `bundle install` so I change the Gemfile to reflect 1.9.3.

**Comment by tjmcs** [comment]
We haven't been doing any testing with JRuby for the past few months to tell you the truth; our internal deployments (and our Travis CI based tests) all use the Docker container we build for Hanlon now, so we hadn't noticed that we added this native dependency to Hanlon when we recently modified Hanlon to use either `ifconfig` (if it is available) or `ip` (if `ifconfig` cannot be found on the local system) to pick the IP address and subnet masks to use in Hanlon (when the initial configuration of Hanlon is being done).

This is actually a continuation of a more involved question around whether or not we should continue to support JRuby as a platform for Hanlon (and the deployment of Hanlon as a WAR file). I've posted the question on both the mailing list and the Gitter page for the project. Hopefully we'll have some discussion in one or both of those places, but I'd welcome a similar discussion in this issue's comments. There are several other issues out there that all relate back to our attempts to support WAR file deployments of Hanlon, and I'm curious how many of our users are actually taking advantage of this feature (and would miss it if it were gone)...

Thoughts from anyone else?
